dotnet/src/Connectors/Connectors.AssemblyAI.UnitTests/Services/AssemblyAIAudioToTextServiceTests.cs

```diff
@@ -28,17 +29,156 @@ public sealed class AssemblyAIAudioToTextServiceTests : IDisposable
         $$"""
           {
             "id": "{{TranscriptGuid}}",
+            "language_model": "assemblyai_default",
+            "acoustic_model": "assemblyai_default",
+            "language_code": "en_us",
+            "status": "queued",
```

Comment:
Not necessary but I suggest moving json contents to TestData folder and load from File like in other connectors.

Reply:
Done